docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,66 @@
+  * For :ref:`WAIT_FOR_FIRST_PASSING_HC<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.EndpointWarmingPolicy.WAIT_FOR_FIRST_PASSING_HC>` endpoint warming policy, if host's health status changes to anything rather than `Healthy`.
+
+Below is example of how requests would be distributed across hosts with Round Robin Loadbalancer, slow start window of 10 seconds, `NO_WAIT` ednpoint warming policy and 0.5 time bias.
+Host H1 has statically configured initial weight of X and host H2 weight of Y, the actual numerical values are of no significance for this example.
+
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| Timestamp   | Event          | H1 in slow | H2 in slow | H1 LB     | H2 LB    | LB decision |
+|             |                | start      | start      | weight    | weight   |             |
++=============+================+============+============+===========+==========+=============+
+| 1           |  H1 create     |    YES     |     --     |   0.5X    |    --    |     --      |
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 11          |  H2 create     |     NO     |    YES     |    X      |   0.5Y   |     --      |
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 12          | LB select host |     NO     |    YES     |    X      |   0.5Y   |     H1      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 13          | LB select host |     NO     |    YES     |    X      |   0.5Y   |     H1      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 14          | LB select host |     NO     |    YES     |    X      |   0.5Y   |     H1      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 15          |LB select host  |     NO     |    YES     |    X      |   0.5Y   |     H2      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 22          | LB select host |     NO     |     NO     |    X      |    Y     |     H1      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+| 23          | LB select host |     NO     |     NO     |    X      |    Y     |     H2      | 
++-------------+----------------+------------+------------+-----------+----------+-------------+
+
+
+.. _arch_overview_load_balancing_slow_start_endpoint_warming_policy_types:
+
+Endpoint warming policy types
+-----------------------------
+
+Endpoint warming policy defines conditions for host to enter slow start mode.
+
+No Wait
+^^^^^^^
+
+If configured, host would enter slow start immediately.
+
+Wait For First Passing Healthcheck
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If configured, host would enter slow start upon having passed an active healthcheck.
```

Comment:
I think this can be moved back to the API docs, assuming we still need the policy after discussion.